### PR TITLE
[NP-613] Add CRUNCH Lab menu item and enhance CRUNCH Lab

### PR DESCRIPTION
**Pull request by KernelDeimos**
- Adds better views of Capability and UserCapabilityJunction inside CRUNCH Lab (which uses the graph view)
- Adds a CRUNCH Lab menu item `admin.crunchlab`

### CODE REVIEW

**Comment by KernelDeimos** [comment]
Screenshot of how this looks with the new capability graph node views:

Screen Shot 2020-07-15 at 11 35 28 PM

File: src/foam/u2/crunch/lab/CapabilityGraphNodeView.js [hunk 1]
```diff
@@ -0,0 +1,125 @@
+foam.CLASS({
```
Comment (jlhughes) [inline src/foam/u2/crunch/lab/CapabilityGraphNodeView.js:7]: license header.

File: src/foam/u2/svg/graph/IdPropertyPlacementPlanDecorator.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
+foam.CLASS({
```
Comment (jlhughes) [inline src/foam/u2/svg/graph/IdPropertyPlacementPlanDecorator.js:7]: license header.

File: src/foam/u2/svg/graph/ZoomedOutFObjectGraphNodeView.js [hunk 3]
```diff
@@ -0,0 +1,53 @@
+foam.CLASS({
```
Comment (jlhughes) [inline src/foam/u2/svg/graph/ZoomedOutFObjectGraphNodeView.js:7]: license header.

File: src/foam/u2/crunch/lab/CapabilityGraphNodeView.js [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+      class: 'Array',
+      factory: () => [0, 0]
+    }
+  ],
+
+  methods: [
+    function initE() {
+      var self = this;
+      var dims = {
+        x: '' + (this.position[0] - 15),
```
Comment (jlhughes) [inline src/foam/u2/crunch/lab/CapabilityGraphNodeView.js:71]: Can the 15, 30 be properties on the view?

**Pull request merged**